### ISSUE TITLE
Xp duplicate

### ISSUE THREAD

**Issue by SPMP2** [Heisenburger69/BurgerSpawners#125]
On breaking monsterspawner, this give xp but players can place monster spawner and break it to infinity and earn infinity xp

**Comment by varunbln** [comment]
You can disable spawner xp drops via config I think

**Comment by xodamcm** [comment]
No, confing doesn’t allow to desactivate spawner xp (breaking mob spawner block = xp)

**Comment by silliershark** [comment]
Hi SPM disable ur auto XP plugin

**Comment by xodamcm** [comment]
Hi, it's already disable. but same

**Comment by silliershark** [comment]
Btw that not problem this plugin is ur auto XP block

**Comment by xodamcm** [comment]
That is problem for me, that suck i don't need an auto xp block i need normally spawner not full xp with place/break place break i need players kill mobs are you stupid?
